spec/models/comment_spec.rb

```diff
@@ -1,5 +1,15 @@
 require 'rails_helper'
```

Comment:
- [ ] It would be better if we add instructions about how to run the test in the README file. Currently, `rspec spec` or `rspec` not working. I had to prepend `bundle exec` to it before it worked

Reply:
Thank you for this, however, `rspec` works fine for me and i have added it to the README file just now.

![Screenshot from 2022-01-29 20-12-37](https://user-images.githubusercontent.com/58692590/151674403-1a15d66e-a5ff-4b93-bf1c-c014243c8735.png)